src/routes/boot.ts

```diff
@@ -195,16 +195,17 @@ const getSquads = async (
         { role },
         { memberPostingRank },
       );
-      // we only send posting permissions from boot to keep the payload small
-      const postingPermissions = permissions.filter(
-        (item) => item === SourcePermissions.Post,
+      // we only send posting and moderation permissions from boot to keep the payload small
+      const essentialPermissions = permissions.filter(
+        (item) =>
+          item === SourcePermissions.Post ||
+          item === SourcePermissions.ModeratePost,
```

Comment:
Why are we changing this? If we are Moderator we can also post, right?

Reply:
But moderator can always bypass which you already know based on currentUser when posting.
I also don't think we need this change.

Reply:
@capJavert / @ilasw  I guess since we decided to handle deciding if it's moderated post on frontend, they need it in boot, as we have flows (context -> share to squad) that rely soley on boot squads now.

(Just FYI, as I struggled to understand the case at first)
Thanks @AmarTrebinjac for explaining the edge-case your facing.

Reply:
ok from my side 👍

Reply:
The TL:DR is basically: 

Post flows use the squads provided by boot, and not the source query